README.md

```diff
@@ -93,9 +98,15 @@ docker run -d -p 5000:5000 --env-file .env todo-app:prod
 ```
 ## DEVELOPMENT RUN
 ```bash
-docker run -d -p 5000:5000 --env-file .env \
---mount type=bind,source=$(pwd)/todo_app,target=/srv/www todo-app:dev 
+docker run -d --name developmentImage -p 5000:5000 --env-file .env --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:dev
+
+Instigate tests: docker exec -it developmentImage "pytest" "--disable-pytest-warnings" "/srv/www/todo_app/tests"
 
 For docker compose, please use: docker-compose -p DevOps up --detach
 ```
 
+## TEST RUN
+```bash
+docker run -d --name TravisCIImage -p 5000:5000 --env-file .env --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:test
```

Comment:
Is there any reason to expose the port here? Answer might be yes if you want your host system to run tests against the container environment, but worth asking whether that's actually something you want